### PR TITLE
Basic structure of Navbar in frontend

### PR DESCRIPTION
**Pull request by ImShubhK**

### CODE REVIEW

**Review by PatelNeelMahesh — APPROVED** [review]
You can merge pull request.

**Comment by PatelNeelMahesh** [comment]
Add description for the PR from next time.

File: README.md [hunk 1]
```diff
@@ -1,3 +1,36 @@
 # TournaTrack
+    
 
-created by shubham
\ No newline at end of file
+  # Back-end
+[] NodeJs - setting up a project with package.json
```
Comment (aditya-190) [inline README.md:5]: Pls Tick the checkboxes for which the topic are covered until now.

File: backend/index.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+const express = require('express');
+const app = express();
+
+const mongoose = require('mongoose')
+
+mongoose.set("strictQuery",false);
+app.use('/',(req,res)=>{
+    res.send("xfdfd")
```
Comment (aditya-190) [inline backend/index.js:8]: Is this for Testing only?

File: backend/package.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
+{
+  "name": "backend",
```
Comment (aditya-190) [inline backend/package.json:2]: Change the name to something like -> <your_project_name>-backend

File: .gitignore [hunk 4]
```diff
@@ -0,0 +1 @@
+node_modules
```
Comment (aditya-190) [inline .gitignore:1]: Delete this gitignore file. It is redundant

File: client/README.md [hunk 5]
```diff
@@ -0,0 +1,70 @@
+# Getting Started with Create React App
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
```
Comment (aditya-190) [inline client/README.md:3]: Update the README to have a basic information about your project.

File: client/package.json [hunk 6]
```diff
@@ -0,0 +1,44 @@
+{
+  "name": "frontend",
```
Comment (aditya-190) [inline client/package.json:2]: Use same naming convection here.

File: client/package.json [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@chakra-ui/react": "^2.5.1",
+    "@emotion/react": "^11.10.6",
+    "@emotion/styled": "^11.10.6",
+    "@testing-library/jest-dom": "^5.16.5",
```
Comment (aditya-190) [inline client/package.json:9]: Add testing libraries under devDependencies

File: client/public/manifest.json [hunk 8]
```diff
@@ -0,0 +1,25 @@
+{
+  "short_name": "React App",
+  "name": "Create React App Sample",
```
Comment (aditya-190) [inline client/public/manifest.json:3]: Write name of your projects.

File: client/src/App.js [hunk 9]
```diff
@@ -0,0 +1,29 @@
+import { Route, Routes } from 'react-router-dom';
+import './component/Navbar';
+import './App.css'; import React from 'react';
+import Navbar from './component/Navbar';
+import Signup from './component/Signup';
+import Home from './component/Home';
+import Register from './component/Register';
+import Detail from './component/Detail';
+import PlayerRegister from './component/PlayerRegister';
+console.log("hit")
```
Comment (aditya-190) [inline client/src/App.js:10]: Comment out or remove the console.log before pushing any code.

File: client/src/App.js [hunk 10]
```diff
@@ -0,0 +1,29 @@
...
+function App() {
+  return (
+    <div className="App">
+      
+     
+      
+       <Routes>
+        <Route path='/' element={<Home/>}/>
+        <Route path='/Register' element={<Register />} />
```
Comment (aditya-190) [inline client/src/App.js:19]: Try to use small case path for routing purpose.

File: q [hunk 11]
```diff
@@ -0,0 +1,39 @@
+[33mcommit 1a39a97c21fe66e13e88016a8f6c07db8fef58d9[m[33m ([m[1;36mHEAD -> [m[1;32mmain[m[33m, [m[1;31morigin/main[m[33m, [m[1;31morigin/HEAD[m[33m)[m
```
Comment (aditya-190) [inline q:1]: Remove this file. It is redundant

**Review by aditya-190 — CHANGES_REQUESTED** [review]
Please make the required changes before merging. Everything else LGTM.

File: README.md (hunk 1 above)
Comment (ImShubhK) [reply README.md:5]: Done

File: backend/index.js (hunk 2 above)
Comment (ImShubhK) [reply backend/index.js:8]: Yes

File: .gitignore (hunk 4 above)
Comment (ImShubhK) [reply .gitignore:1]: Done

File: client/src/App.js (hunk 9 above)
Comment (ImShubhK) [reply client/src/App.js:10]: Done

File: client/src/App.js (hunk 10 above)
Comment (ImShubhK) [reply client/src/App.js:19]: Done

File: client/src/component/Navbar.js [hunk 12]
```diff
@@ -31,7 +32,11 @@ function Navbar() {
          
         </li>
         <li className="nav-item">
-         <button>SignUp</button>
+        <Link to='/Signup'>
```
Comment (aditya-190) [inline client/src/component/Navbar.js:35]: Update the existing routes to smallCase

**Pull request merged**